src/index.tsx

```diff
@@ -21,22 +23,38 @@ export const ListsApp: FC<IListsApp> = (props) => {
         <Route
           path={path}
           exact
-          component={ListPage}
+          render={() => (
+            <IfPermission perm="lists.collection.get">
```

Comment:
Let's keep all permissions names in separate enum and reuse everywhere as a constant